libs/network/socketserver.c

```diff
@@ -479,7 +519,7 @@ bool socketClientConnectionsIdle(JsNetwork *net) {
       if (isHttp)
         hadHeaders = jsvGetBoolAndUnLock(jsvObjectGetChild(connection,HTTP_NAME_HAD_HEADERS,0));
       else
-        hadHeaders = true;
+        hadHeaders = (socketType&ST_TYPE_MASK)!=ST_UDP;
```

Comment:
I haven't run this, but I'd have thought hadHeaders should be true even for UDP? If there aren't any headers it's expecting it should be true I think.